ar.m3u

MoonSyria opened the pull request:
Update the Arabic language channelss' links and re-sort channels.

MoonSyria committed:
Add files via upload

MoonSyria committed:
Delete Aflam.m3u8

MoonSyria committed:
Update lb.m3u

Update 
Aghani channel

MoonSyria committed:
Update iq.m3u

Delete Afaq TV (Broken link)

MoonSyria committed:
Update ar.m3.u

Update many broken links
Sorted many channels and create a new TV group (MBC)

freearhey reviewed (changes requested):
Sorry, but there's been a mistake. "AR" stands for "Argentina" not "Arabic".

MoonSyria committed:
Update Arabic.m3u to arabic.m3u

1- Updated the name because there is an old channel list with the same name
2- Fix broken links
a- Al Arabiya Al Hadath ARB
b- Al Arabiya ARB
3- Fix EPG

MoonSyria committed:
Update Arabic.m3u

Updated links for Aloula

MoonSyria commented:
Sorry I did not notice for Ar belong to Argentine
I updated it to Arabic.m3u
I updated Arabic list and fixed EPG and some broken links

MoonSyria committed:
Update Arabic.m3u

freearhey commented:
You don't understand, the channels in this repository are sorted by country. And the files are named according to [ISO 3166](https://en.wikipedia.org/wiki/ISO_3166) country codes. You can read more here: https://github.com/iptv-org/iptv/blob/master/.github/CONTRIBUTING.md

MoonSyria committed:
Update Arabic.m3u

MoonSyria committed:
Update Arabic.m3u

update Al Arabya and Al Arabya Alhadath
